01-JS-DOM-APIs/Exercises1_2/js/main.js

```diff
@@ -0,0 +1,12 @@
+window.onload = function(a)
+{
+    hidden();
+    document.getElementById("btn").addEventListener("click", function showAlert(){
```

Comment:
* Create a function that showcases an alert message when called.

* Attach a click event to the created button which calls the function previously created.

First declare your function and then add it to the event.